cmd/emp/dynos.go

```diff
@@ -58,6 +58,9 @@ func listDynos(w io.Writer) {
 func listDyno(w io.Writer, d *heroku.Dyno) {
 	listRec(w,
 		d.Name,
+		d.Id,
+		d.ContainerInstanceID,
+		d.EC2InstanceID,
```

Comment:
Let me know if I should modify https://github.com/remind101/empire/blob/master/pkg/heroku/gen/schema.json instead and generate the change